### DISCUSSION TITLE
Use Midi Instruments On Jamuls

### DISCUSSION THREAD

**Discussion by klepto84** [jamulussoftware/jamulus#1236] [Technical Support & Help]
First of all, THANK YOU to everyone who has contributed to Jamulus.  This is amazing and life-changing. Could anyone point me to a resource that will explain how to use Jamulus to play midi instruments with Ableton Live 11?
I am using Windows 10, Focusrite Scarlett 2i2 3rd Gen., /Ableton Live 11 Standard, Novation Launchkey 49 Mk2 with Spitfire AudioBBC Orchestra and Native Instruments Komplete. Jamulus works well for me, with mic and bass/guitar/digital piano running into the audio interface, but I can't use my midi instruments or Ableton effects.  Thanks.  Also is there a "checklist" for troubleshooting common problems?  I've had zero issues with Jamulus which were not self-inflicted (i.e. didn't hit phantom power), but some of my friends are having difficulty setting up.

**Comment by ann0see** [comment]
I think you can route Audio from your DAW to Jamulus with Voicemeeter Banana (not much experience with this) or JACK. 

There is a troubleshooting page on jamulus.io (look at the sidebar there). Also most setup issues I know of are related to ASIO4ALL. Have you tried ASIO4ALL 2.15 Beta already and linked them the setup instructions from the website?
